util/config/schema.py

```diff
@@ -1260,5 +1260,10 @@
             "description": "Whitelisted users to exclude when FEATURE_RESTRICTED_USERS is enabled",
             "x-example": ["devtable"],
         },
+        "FEATURE_NOTIFY_NEW_PUSH_VULNS": {
```

Comment:
Can you rename this `FEATURE_SECURITY_SCANNER_NOTIFY_ON_NEW_INDEX` or something similar to keep it consistent with the other flags?